api/src/main/java/org/asynchttpclient/Request.java

```diff
@@ -112,6 +113,13 @@
     InputStream getStreamData();
 
     /**
+     * Return the current request's body as a Reactive Stream Publisher.
+     *
+     * @return a Reactive Stream Publisher representation of the current request's body.
```

Comment:
Sadly, I don't see any convenient solution for not having core API depend on reactive-streams.